core/src/main/java/io/questdb/cairo/TxReader.java

```diff
@@ -209,6 +214,7 @@ public boolean unsafeLoadAll() {
             final long prevPartitionTableVersion = this.partitionTableVersion;
             this.partitionTableVersion = getLong(TableUtils.TX_OFFSET_PARTITION_TABLE_VERSION_64);
             this.columnVersion = unsafeReadColumnVersion();
+            this.truncateVersion = getLong(TableUtils.TX_OFFSET_TRUNCATE_VERSION_64);
```

Comment:
We need to update `dumpTo()` method in this class as well to make sure that the truncate version is included into snapshots. Also it'd be nice to update _txn file's contents verification in `SnapshotTest`.

Reply:
done